js/modules/k6/experimental/browser/rootmodule.go

```diff
@@ -0,0 +1,49 @@
+package browser
+
+import (
+	"errors"
+	"strconv"
+
+	xk6browser "github.com/grafana/xk6-browser/browser"
+	"go.k6.io/k6/js/common"
+	"go.k6.io/k6/js/modules"
+)
+
+type (
+	// RootModule is the global module instance that will create module
+	// instances for each VU.
+	RootModule struct {
+		rm *xk6browser.RootModule
+	}
+)
+
+func New() *RootModule {
+	return &RootModule{
+		rm: xk6browser.New(),
+	}
+}
+
+func (r *RootModule) NewModuleInstance(vu modules.VU) modules.Instance {
+	env := vu.InitEnv()
+
+	throwError := func() {
+		msg := "To run browser tests set env var K6_BROWSER_ENABLE_RUN=true"
+		if m, ok := env.LookupEnv("K6_BROWSER_ENABLE_RUN_MSG"); ok && m != "" {
+			msg = m
+		}
+
+		common.Throw(vu.Runtime(), errors.New(msg))
+	}
+
+	vs, ok := env.LookupEnv("K6_BROWSER_ENABLE_RUN")
```

Comment:
```suggestion
	vs, ok := env.LookupEnv("K6_BROWSER_ENABLED")
```

This sounds a bit better? :thinking: